docs/root/start/quick-start/run-envoy.rst

```diff
@@ -89,8 +89,8 @@ Envoy will parse the config file according to the file extension, please see the
 
    .. tab:: System
 
-      To start Envoy as a system daemon :download:`download the demo configuration <_include/envoy-demo.yaml>`, and start
-      as follows:
+      To start Envoy as a system daemon :download:`download the demo configuration <_include/envoy-demo.yaml>`. Confirm that the downloaded file has
+      the `.yaml` extension (and not `.yml`) and start as follows:
```

Comment:
im not sure this is necessary - what problem is addressing ?

Reply:
On my macbook (latest Catalina and Chrome) when I clicked on that link, it saved the file with a `.yml` extension. Not sure why. Without realizing that, when I ran envoy with it (using zsh tab completion for the file name) it exited with this error. 

```
[2021-07-21 05:23:43.653][397997][critical][main] [source/server/server.cc:113] error initializing configuration 'envoy-demo.yml': Unable to parse JSON as proto (INVALID_ARGUMENT:Unexpected token.
```
It wasn't obvious from the error that the problem was with the file extension. I found that out via web search. It might be good if the config file parsing code works with either extension. Perhaps there's some background on this I don't know 🙂. So I was thinking this suggestion might save someone else some time during the initial try/quickstart.

Reply:
The sample config is saved with the correct `.yaml` extension a different macbook running the latest macOS Big Sur. Here are the screenshots. So may be more of a corner case I ran into. 
![bigsur-Screen Shot 2021-07-21 at 5 55 15 AM](https://user-images.githubusercontent.com/3322496/126494576-3546e420-a56a-41ed-b9c4-90b81158d2df.png)
![catalina-Screen Shot 2021-07-21 at 5 53 02 AM](https://user-images.githubusercontent.com/3322496/126494584-26dc9038-06b3-4751-9a96-221753b384e4.png)